src/order-book/api.ts

```diff
@@ -410,10 +409,11 @@ export class OrderBookApi {
    * @returns The response from the API.
    */
   private fetch<T>(params: FetchParams, contextOverride: PartialApiContext = {}): Promise<T> {
-    const { chainId, env } = this.getContextWithOverride(contextOverride)
+    const { chainId, env, backoffOpts: _backoffOpts } = this.getContextWithOverride(contextOverride)
     const baseUrl = this.getApiBaseUrls(env)[chainId]
-    const backoffOpts = this.context.backoffOpts || DEFAULT_BACKOFF_OPTIONS
+    const backoffOpts = _backoffOpts || DEFAULT_BACKOFF_OPTIONS
+    const rateLimiter = contextOverride.limiterOpts ? new RateLimiter(contextOverride.limiterOpts) : this.rateLimiter
```

Comment:
Doesn't this need to be fetched from `this.getContextWithOverride(contextOverride)` like `backoffOpts`?

Reply:
No, here we use `this.rateLimiter` by default and create a new only only when `limiterOpts` is set in `contextOverride`